### PR TITLE
fill out pgbouncer docs

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

File: content/integrations/pgbouncer.md [hunk 1]
```diff
@@ -3,17 +3,49 @@ title: Datadog-PGBouncer Integration
...
 Connect your PGBouncer to Datadog in order to:
 
 * Visualize your pools of connections.
 * Monitor the traffic between PostgreSQL and your applications.
+* Be notified about pgbouncer failovers and events.
+
+
+### Configuration
+
+To configure the PGBouncer integration, copy `pgbouncer.yaml.example` to `pgbouncer.yaml` and mak the appropriate changes.
```
Comment (kmshultz) [inline content/integrations/pgbouncer.md]: typo: s/mak/make

File: content/integrations/pgbouncer.md [hunk 2]
```diff
@@ -3,17 +3,49 @@ title: Datadog-PGBouncer Integration
...
+    Checks
+    ======
+
+        pgbouncer
+        -----------
+          - instance #0 [OK]
+          - Collected 39 metrics, 0 events & 7 service checks
 ### Metrics
 
 <%= get_metrics_from_git() %>
```
Comment (kmshultz) [inline content/integrations/pgbouncer.md]: We should document `pgbouncer.can_connect` in a Service Checks section, too.

Comment (technovangelist) [inline content/integrations/pgbouncer.md]: do you have an example of a service checks section?

File: content/integrations/pgbouncer.md [hunk 3]
```diff
@@ -3,17 +3,54 @@ title: Datadog-PGBouncer Integration
...
+          - instance #0 [OK]
+          - Collected 39 metrics, 0 events & 1 service check
 ### Metrics
 
 <%= get_metrics_from_git() %>
+
+### Service Checks
+
+**pgbouncer.can_connect**
+: Agent is able to connect to the pgbouncer instance
```
Comment (kmshultz) [inline content/integrations/pgbouncer.md]: This should be clear about check status, i.e. "Returns CRITICAL if the Agent is unable to connect to the pgbouncer instance".

**Pull request merged**